lib/internal/Magento/Framework/App/Response/Http/FileFactory.php

```diff
@@ -76,7 +76,11 @@ public function create(
             ->setHeader('Pragma', 'public', true)
             ->setHeader('Cache-Control', 'must-revalidate, post-check=0, pre-check=0', true)
             ->setHeader('Content-type', $contentType, true)
-            ->setHeader('Content-Length', $contentLength === null ? strlen($content) : $contentLength, true)
+            ->setHeader(
+                'Content-Length',
+                $contentLength === null ? strlen($this->getFileContent($content)) : $contentLength,
```

Comment:
Comment on code readability:
Since you use this method twice (line 81 and 95), why not to put the result in a variable?

Reply:
Agree, the mentioned approach improves the readability, thanks.